SEC/Equities/EquityInstruments.rdf

```diff
@@ -406,12 +420,73 @@
 		<skos:definition xml:lang="en">position in an equity instrument</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-sec-eq-eq;EquityRedemptionProvision">
```

Comment:
Should this not be named specific to PreferredEquity? Can you redeem any other type of equity?

Reply:
Yes - restricted common shares.